requirements.txt

```diff
@@ -328,11 +328,11 @@ click==8.1.8 ; python_version >= "3.11" and python_full_version <= "3.12.3" \
 colorama==0.4.6 ; python_version >= "3.11" and python_full_version <= "3.12.3" and (platform_system == "Windows" or sys_platform == "win32") \
     --hash=sha256:08695f5cb7ed6e0531a20572697297273c47b8cae5a63ffc6d6ed5c201be6e44 \
     --hash=sha256:4f1d9991f5acc0ca119f9d443620b77f9d6b33703e51011c16baf57afb285fc6
-flet-desktop-light==0.26.0 ; sys_platform == 'linux' and python_version >= "3.11" and python_full_version <= "3.12.3" \
```

Comment:
Whats going on here? That `sys_platform` is important, since the flet-desktop-light is only available for linux and is important to make it work in the flatpak. Did you notice any issues with it?

Reply:
Not sure, I didnt check what automatically updating the deps did
I just "make install" and export ig. I didnt test on anything but ubuntu

Reply:
alr, Ive dropped that commit, not sure why poetry did that by default...